test/sqpoll-disable-exit.c

```diff
@@ -82,10 +82,10 @@ static long syz_io_uring_setup(volatile long a0, volatile long a1,
 {
   uint32_t entries = (uint32_t)a0;
   struct io_uring_params* setup_params = (struct io_uring_params*)a1;
-  void* vma1 = (void*)a2;
```

Comment:
It's changes like this I mean when I don't do unrelated formatting changes. And all of these test changes should be done prior to adding this sanitizer commit. Yes checking for the defined would be a no-op before that - but that's fine, because there's no sanitizer anyway.